specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-08-25/workbooks_API.json

```diff
@@ -0,0 +1,808 @@
+          "description": "Resource tags"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Metadata describing a workbook for an Azure resource.",
+          "$ref": "#/definitions/WorkbookPropertiesUpdateParameters"
+        }
+      },
+      "type": "object",
+      "description": "The parameters that can be provided when updating workbook properties properties."
+    },
+    "WorkbookPropertiesUpdateParameters": {
+      "description": "Properties that contain a workbook for PATCH operation.",
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The user-defined name (display name) of the workbook."
+        },
+        "serializedData": {
+          "type": "string",
+          "description": "Configuration of this particular workbook. Configuration data is a string containing valid JSON"
+        },
+        "category": {
+          "type": "string",
+          "description": "Workbook category, as defined by the user at creation time."
+        },
+        "tags": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A list of 0 or more tags that are associated with this workbook definition"
+        },
+        "description": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "The description of the workbook."
+        },
+        "revision": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "The unique revision id for this workbook definition"
+        }
+      }
+    },
+    "WorkbookError": {
+      "description": "Error response.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorDefinition",
+          "description": "The error details."
+        }
+      }
+    },
+    "ErrorDefinition": {
+      "description": "Error definition.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Service specific error code which serves as the substatus for the HTTP error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Description of the error.",
+          "type": "string",
+          "readOnly": true
+        },
+        "innererror": {
+          "description": "Internal error details.",
+          "items": {
+            "$ref": "#/definitions/InnerErrorTrace"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "InnerErrorTrace": {
+      "description": "Error details",
+      "type": "object",
+      "properties": {
+        "trace": {
+          "description": "detailed error trace",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "WorkbookManagedIdentity": {
```

Comment:
>WorkbookManagedIdentity [](http://example.com/codeflow?start=5&length=23)

This needs to be updated since it's missing the userAssignedIdentities field, something like this:

  "IdentityData": {
    "required": [
      "type"
    ],
    "type": "object",
    "description": "Identity for the resource.",
    "properties": {
      "principalId": {
        "readOnly": true,
        "type": "string",
        "description": "The principal ID of resource identity."
      },
      "tenantId": {
        "readOnly": true,
        "type": "string",
        "description": "The tenant ID of resource."
      },
      "type": {
        "type": "string",
        "description": "The type of managed identity used. The type 'SystemAssigned, UserAssigned' includes both an implicitly created identity and a set of user-assigned identities. The type 'None' will remove any identities.",
        "enum": [
          "SystemAssigned",
          "None",
          "UserAssigned",
          "SystemAssigned, UserAssigned"
        ],
        "x-ms-enum": {
          "name": "ResourceIdentityType",
          "modelAsString": true
        }
      },
      "userAssignedIdentities": {
        "description": "The list of user-assigned identities associated with the resource. The user-assigned identity dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'.",
        "type": "object",
        "additionalProperties": {
          "$ref": "#/definitions/UserIdentity"
        }
      }
    }
  },
  "UserIdentity": {
    "type": "object",
    "description": "A resource identity that is managed by the user of the service.",
    "properties": {
      "principalId": {
        "description": "The principal ID of the user-assigned identity.",
        "type": "string",
        "readOnly": true
      },
      "clientId": {
        "description": "The client ID of the user-assigned identity.",
        "type": "string",
        "readOnly": true
      }
    }
  },